src/runtime/trace/trace.go

```diff
@@ -11,3 +12,29 @@ func Start(w io.Writer) error {
 }
 
 func Stop() {}
+
+func NewTask(pctx context.Context, taskType string) (ctx context.Context, task *Task) {
+	return context.TODO(), nil
+}
+
+type Task struct{}
+
+func (t *Task) End() {}
+
+func Log(ctx context.Context, category, message string) {}
+
+func Logf(ctx context.Context, category, format string, args ...any) {}
+
+func WithRegion(ctx context.Context, regionType string, fn func()) {}
```

Comment:
I don't think this is correct. The documentation says:

> WithRegion starts a region associated with its calling goroutine, runs fn, and then ends the region.

So even if we don't implement tracing, I think `WithRegion` should at least call `fn`.

Reply:
Good call, I've adjusted the function to call the passed in `fn`, thanks!